pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -46,14 +46,29 @@ type AccountWatcher struct {
 func initialize(client client.Client, watchInterval time.Duration) *AccountWatcher {
 	log.Info("Initializing the totalAccountWatcher")
 
+	cm := &corev1.ConfigMap{}
+	err := client.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if err != nil {
+		log.Error(err, "Failed to get configmap")
+	}
+
+	awsRegion := awsv1alpha1.AwsUSEastOneRegion
+	fr, ok := cm.Data["fedramp"]
+	if ok {
+		frBool, _ := strconv.ParseBool(fr)
```

Comment:
is it ok to ignore the error here?

Reply:
good catch. i was assuming that successfully reading `cm.Data["fedramp"]` would ensure that `fr` is a bool but we shouldn't make that assumption

Reply:
actually, if `cm.Data["fedramp"]` returns any value in `fr`, then `strconf.ParseBool()` _should_ return either true or false in `frBool`

but there's no reason to tempt fate. i'll check the error code anyway :)